js/renderingPicture.js

```diff
@@ -0,0 +1,37 @@
+import {createPhotos} from './mockPhotos.js';
+
+const dataCreatePicture = createPhotos();
+const SAMPLE = document.querySelector('#picture').content.cloneNode(true);
+const PICTURES = document.querySelector('.pictures');
+const fragment = document.createDocumentFragment();
+
+const createPictures = (data, sample) =>{
```

Comment:
Конкретно эта функция у тебя создает 1 плитку фотографии/картинки, не нужно множественное число.
И любая функция в js замыкание -- она видет все определенные переменные на момент определения, нет в данном случае необходимости добавлять sample в качестве аргумента: она же определена выше, значит внутри этой функции будет доступ к этой внешней переменной
```suggestion
const createPicture = (data) =>{
```

Reply:
+